loadtest/user/userentity/websocket.go

```diff
@@ -18,6 +18,22 @@ const (
 	maxWebsocketFails             = 7
 )
 
+func (ue *UserEntity) handlePostEvent(ev *model.WebSocketEvent) error {
+	var post *model.Post
+	if err := json.Unmarshal([]byte(ev.Data["post"].(string)), &post); err != nil {
+		return err
+	}
+
+	evType := ev.EventType()
+	if evType == model.WEBSOCKET_EVENT_POSTED || evType == model.WEBSOCKET_EVENT_POST_EDITED {
```

Comment:
A switch-case is preferred here to avoid the repetetive "evType ==" checks.

Reply:
It's two cases/branches. I usually start considering a `switch` if I am dealing with at least 3.  
I guess it depends whether it's more annoying the repeated comparison or the added syntax.  
To me it's the latter but totally understand your reasoning :+1:

Reply:
Yes I have the threshold of 3 too. I actually count the comparisons also. So that way, it does cross 3. Thats why I suggested. Not a big deal. Feel free to take a call.

Reply:
Thinking more about it I believe the switch works fine in this particular case since the block would only consist of a single call to the store function. Also, now that I know you can handle multiple cases on the same `case` line as you suggested above it does indeed look like the cleanest approach.